lib/opentelemetry/trace/id_generator.lua

```diff
@@ -4,39 +4,22 @@ local bit = require 'bit'
 local tohex = bit.tohex
 local fmt = string.format
 local random = util.random
+local FFFFFFFF = 4294967295 -- FFFFFFFF in hexadecimal is 4294967295 in decimal
```

Comment:
Great Job! I think we can use `0xffffffff` directly.

Reply:
Re-ran benchmarks, still looks faster: `5m new ids: 0.118254 seconds.`